### PR TITLE
Reduce yaml files for ha_textmode_minimal_base

### PR DESCRIPTION
**Pull request by hjluo**
##
### Reduce yaml files for ha_textmode_minimal_base

- Related ticket: 
   - https://progress.opensuse.org/issues/156034
-  Related MR:
   - https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/244
- Needles: 
   - N/A
- Verification run: 
   - [**VR**](https://openqa.suse.de/tests/overview?version=15-SP6&build=hjluo%2Fos-autoinst-distri-opensuse%23reduce_ha_textmode_minimal&distri=sle)
 ##

### CODE REVIEW

File: schedule/yast/sle/flows/default_s390x_kvm.yaml [hunk 1]
```diff
@@ -39,8 +39,7 @@ installation:
   - installation/launch_installation
   - installation/confirm_installation
   - installation/performing_installation/perform_installation
-stop_timeout_system_reboot:
-  - installation/performing_installation/stop_timeout_system_reboot_now
+stop_timeout_system_reboot: []
```
Comment (jknphy) [inline schedule/yast/sle/flows/default_s390x_kvm.yaml:42]: you cannot do that without breaking many test suite, please coordinate with @lemon-suse he is working in similar solution, which is create another yaml default for textmode I think.

Comment (hjluo) [reply schedule/yast/sle/flows/default_s390x_kvm.yaml]: but why don't use `schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml`? it's still [**works**](https://openqa.suse.de/tests/14759341#settings) and we don't need `system_preparation` in PR #19622

Comment (lemon-suse) [reply schedule/yast/sle/flows/default_s390x_kvm.yaml]: Reuse the schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml now. For system_preparation, not a must to remove it, and to unify schedule of test suite, we can't simply to overwrite that in test suite yaml.

Comment (hjluo) [reply schedule/yast/sle/flows/default_s390x_kvm.yaml]: @lemon-suse  @jknphy  restore it to it's original and please take a look again.  Thanks!

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**